igvc_gazebo/nodes/ground_truth/GroundTruth.h

```diff
@@ -0,0 +1,25 @@
+// GroundTruth.h
+#ifndef GroundTruth_H
+#define GroundTruth_H
+
+#include <nav_msgs/Odometry.h>
+#include <ros/ros.h>
+#include <tf/transform_broadcaster.h>
+#include <tf/transform_datatypes.h>
+#include <tf/transform_listener.h>
+
+
+class GroundTruth 
+{
+public:
+	ros::Publisher g_ground_truth_pub;
+	nav_msgs::Odometry g_og_pose;
+	ros::Time g_last_estimate;
```

Comment:
Remove the g_ prefix, as these are no longer global variables. Also, these should be private variables.